docs/en/features/local_census.md

```diff
@@ -1,33 +1,36 @@
-# Local Census
+# Local census
 
-To provide to administrator users a way to manage the local census database through the administration panel **Settings > Manage local census**. Currently the only way to manipulate this table records is through the rails console.
+This feature is designed for installations with limited resources that cannot configure and customize their census connection remotely, so we offer them a solution to use a local census instead. For this purpose, administrators are provided with a way to manage the local census database through the administration panel under **Settings > Local census**.
 
-Allow administrators users to manage this table in two different ways:
+## Manage local census
 
-- **Manually**: one by one through a CRUD interface.
-- **Automatically**: through an importation process.
+Administrators can manage this census in two different ways:
 
-## Manually
+* **Manually**: add one by one through a form.
```

Comment:
Reading this sentence, I'm not sure what's added one by one :thinking:. Maybe something like:

```suggestion
* **Manually**: adding one citizen at a time through a form.
```